main/src/cgeo/geocaching/CacheListActivity.java

```diff
@@ -1,5 +1,38 @@
 package cgeo.geocaching;
 
+import android.app.Activity;
```

Comment:
The imports were re-ordered. I'm not sure what's our current style guideline, but this shouldn't be included in this pull request and fixed in another commit for all files if necessary.